doc/doxyfile.in

```diff
@@ -44,14 +44,14 @@ PROJECT_NUMBER         = @VERSION_MAJOR@.@VERSION_MINOR@.@VERSION_PATCH@
 # for a project that appears at the top of each page and should give viewer a
 # quick idea about the purpose of the project. Keep the description short.
 
-PROJECT_BRIEF          = 
+PROJECT_BRIEF          =
```

Comment:
I don't see how it can even _potentially_ cause trouble, unless doxygen parsing was really broken in some way.

And by the way, this file was barely ever touched in the past. Quite coincidentally, the last 2 _big_ changes to it were fc850ed99fc3e52e16c7d9d1bf71daf92cbe5f76 and 8b7ac65aa35dc99a61e8c48f801961e49f12564a, the former removing trailing spaces and the latter introducing them again. If in the future we make sure to remove them before checking in any modified versions of the file, we shouldn't have anything else to talk about if you ask me. :stuck_out_tongue: